esileapclient/v1/node.py

```diff
@@ -21,14 +21,24 @@
 class Node(base.Resource):
 
     detailed_fields = {
-        'name': "Name"
+        'name': "Name",
+        'uuid': "UUID",
+        'owner': "Owner",
+        'offer_uuid': "Offer UUID",
+        'lease_uuid': "Lease UUID",
+        'lessee': "Lessee"
     }
 
     fields = {
-        'name': "Name"
+        'name': "Name",
+        'owner': "Owner",
+        'offer_uuid': "Offer UUID",
+        'lease_uuid': "Lease UUID",
+        'lessee': "Lessee"
     }
 
-    _creation_attributes = ['name']
+    _creation_attributes = ['name', 'uuid', 'owner', 'offer_uuid',
```

Comment:
I'm not sure this variable is needed at all, since we'll never be creating nodes from the esi-leap CLI

Reply:
I thought so, too, but it threw an error when I tried to remove it. I might take another look.

Reply:
Yeah, if I take it out, it throws ```Can't instantiate abstract class Node with abstract methods _creation_attributes```